baseplate/experiments/variant_sets/range_variant_set.py

```diff
@@ -0,0 +1,68 @@
+from .base import VariantSet
+
+
+class RangeVariantSet(VariantSet):
+    """ Variant Set designed to take fixed bucket ranges.
```

Comment:
nit: please follow pep257 guidelines for docstrings. the first line should be a standalone summary and then there should be a blank followed by as much extra docs as you need.